routers/router.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const TaskController = require('../controllers/controller')
+
+const router = express.Router()
+
+router.get('/', TaskController.findAllTasks)
+router.post('/', TaskController.addTask)
+router.patch('/content/:id', TaskController.updateTitle)
+router.patch('/status/:id', TaskController.updateStatus)
```

Comment:
Why did you use `patch` here?

Reply:
because I only want to update 1 field in the database so I think PATCH will be more reasonable than PUT

Reply:
I understand what you want to do here. But we should make the route as simplest as possible. I suggest you using 
```
router.put('/:id/', TaskController.updateTask)
```
with the function updateTask handle updating task's title and status.